installer/tue-install-impl.bash

```diff
@@ -1007,20 +1032,16 @@ function _tue-install-pip-now
 
     if [ -n "$pips_to_install" ]
     then
-        echo -e "Going to run the following command:\n"
-        echo -e "python${pv} -m pip install --use-feature=2020-resolver --use-feature=fast-deps --user $pips_to_install <<< yes\n"
         # shellcheck disable=SC2048,SC2086
-        python"${pv}" -m pip install --use-feature=2020-resolver --use-feature=fast-deps --user $pips_to_install <<< yes || tue-install-error "An error occurred while installing pip${pv} packages."
+        tue-install-pipe python"${pv}" -m pip install --use-feature=fast-deps --user $pips_to_install <<< yes || tue-install-error "An error occurred while installing pip${pv} packages."
```

Comment:
Also remove the --use-feature=fast-deps. The author of the feature himself said it doesn't work.